pkgs/tools/archivers/unzip/default.nix

```diff
@@ -23,13 +22,13 @@ stdenv.mkDerivation {
     ./CVE-2015-7697.diff
     ./CVE-2014-9913.patch
     ./CVE-2016-9844.patch
+    ./cross-cc.patch;
```

Comment:
Actually it is. I changed makefile to only define the c compiler as a fallback: `CC ?= cc`. What I can do is give the patch a better name :)